src/Kdyby/Doctrine/EntityManager.php

```diff
@@ -74,7 +77,7 @@ public function bindTracyPanel(Diagnostics\Panel $panel)
 	/**
 	 * @return \Kdyby\Doctrine\QueryBuilder
 	 */
-	public function createQueryBuilder()
+	public function createQueryBuilder($alias = NULL, $indexBy = NULL)
```

Comment:
We might consider throw an `NotImplementedException` when they're not `=== NULL`